test/helpers/JsonFixtures.scala

```diff
@@ -61,6 +61,36 @@ object JsonFixtures {
       |}""".stripMargin
   }
 
+  def displaySubscriptionPayloadNoSecondary(firstName: JsString,
+                                            lastName:JsString,
+                                            primaryEmail: JsString,
+                                            phone: JsString): String = {
+    s"""
+       |{
+       |  "displaySubscriptionForDACResponse": {
+       |    "responseCommon": {
+       |      "status": "OK",
+       |      "processingDate": "2020-08-09T11:23:45Z"
+       |    },
+       |    "responseDetail": {
+       |      "subscriptionID": "XE0001234567890",
```

Comment:
if this subscription id is the right format and valid it will need to be generated